js/chrisAPI/.babelrc

```diff
@@ -2,7 +2,15 @@
   "presets": [
     [
       "@babel/preset-env", {
-        "useBuiltIns": "usage"
+        "useBuiltIns": "usage",
+        "corejs": "3",
+        "targets": {
+          "node": "8.10",
+          "browsers": [
+            "last 2 versions",
```

Comment:
I'm only slightly apprehensive of this line. But if it means the last two major versions then we might be okay?

Reply:
@zrthxn , We should be good there. Most developers set their version lists using queries like last 2 versions to be free from updating versions manually. Browserslist will use caniuse-lite with Can I Use data for this queries.